event_model/__init__.py

```diff
@@ -71,6 +71,14 @@ def __call__(self, name, doc, validate=False):
             The same name as what was passed in, and a doc that may be the same
             instance as doc, a copy of doc, or a different dict altogether.
         """
+        return self._dispatch(name, doc, validate)
+
+    def _dispatch(self, name, doc, validate):
+        """
+        Dispatch to the method corresponding to the `name`.
```

Comment:
```suggestion
        Dispatch to the method corresponding to the ``name``.
```

Reply:
This method is private and won't show up in sphinx. :-D